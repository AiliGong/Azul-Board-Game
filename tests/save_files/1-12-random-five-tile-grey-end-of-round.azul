#myformat
3
5
1
a
b
LULRBRUURRYLRYLLYBUBUYUYYURYLYRBULLBBBUYRYRLLLUYRYYBLLYRBLRBUYUURRBRUBLUURBBBRBLYYBYULLUBRLRUYBBLYUR
%1
%1
%1
turn 1 L 2
turn 2 U 2
turn 3 R 3
turn 4 L 3
turn 5 B 4
turn 0 R 4
turn 0 Y 5
turn 0 L 3
turn 0 B 4
turn 0 U 5
