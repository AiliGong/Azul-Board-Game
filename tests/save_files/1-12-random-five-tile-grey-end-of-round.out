Factories:
0: F
1: U Y U Y
2: Y U R Y
3: L Y R B
4: U L L B
5: B B U Y

Score for a: 1
Mosaic for a:
1:         . || . . . . . 
2:       . . || L . . . . 
3:     . R R || . . . . . 
4:   . B B B || . . . . . 
5: . . Y Y Y || . . . . . 
broken:

Score for b: 2
Mosaic for b:
1:         . || . . . . . 
2:       . . || U . . . . 
3:     . . . || L . . . . 
4:   . R R R || . . . . . 
5: . . . U U || . . . . . 
broken:
